src/connectors/hits-per-page/__tests__/connectHitsPerPage-test.ts

```diff
@@ -308,27 +343,34 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/hits-per-pa
       ],
     });
 
-    const helper = algoliasearchHelper({}, '', {
+    const searchClient = createSearchClient();
+    const helper = algoliasearchHelper(searchClient, '', {
       hitsPerPage: 20,
     });
     helper.search = jest.fn();
 
-    widget.init({
-      helper,
-      state: helper.state,
-      createURL: state => state,
-    });
+    widget.init!(
+      createInitOptions({
+        helper,
+        state: helper.state,
+        createURL: state => state as any,
```

Comment:
If you want to keep type safety, you can return `JSON.stringify(state)` and parse it line 360.